src/pages/about/story/index.js

```diff
@@ -0,0 +1,169 @@
+import React, { useState, useEffect } from 'react'
+import styled, { css } from 'styled-components'
+import { OurStory } from './_our-story'
+import { Container, Flex, SEO, Show } from 'components/containers'
+import { getWindowWidth, isBrowser } from 'common/utility'
+import Layout from 'components/layout/layout'
+import { localize, WithIntl } from 'components/localization'
+import { Header, Text } from 'components/elements'
+import device, { size } from 'themes/device'
+
+const Background = styled.div`
+    background: var(--color-black);
+    width: 100%;
+    height: 100%;
+`
+
+const StyledContainer = styled(Container)`
+    padding: 12rem 19rem;
+    white-space: nowrap;
+
+    @media ${device.laptopL} {
+        padding: 12rem 4rem;
+    }
+    @media ${device.tabletL} {
+        padding: 5rem 2rem;
+        width: 100%;
+
+        > h1 {
+            font-size: 4.5rem;
+        }
+    }
+`
+const ContentWrapper = styled.div`
+    margin-top: ${(props) => props.margin_top || 'none'};
+    white-space: normal;
+    max-width: 79.8rem;
+
+    @media ${device.mobileL} {
+        max-width: 42.8rem;
+    }
+`
+
+const NavigationWrapper = styled(Flex)`
+    margin: 1.6rem 0;
+`
+
+const Navigation = styled(Flex)`
+    cursor: pointer;
+    margin: 0 2.4rem;
+
+    @media ${device.tablet} {
+        margin: ${(props) => (props.left ? '0 3rem 0 0' : '0 0 0 3rem')};
+    }
+    @media ${device.mobileS} {
+        margin: ${(props) => (props.left ? '0 2rem 0 0' : '0 0 0 2rem')};
+    }
+`
+
+// const Separator = styled.span`
```

Comment:
Let's remove this comment, if not needed.

Reply:
Done